frontend/src/views/File.vue

```diff
@@ -132,14 +141,18 @@ img.product
     <div v-if="response.volatile" class="note volatilenote">
       This is a volatile file. The data in this file may be incomplete and update in real time.
     </div>
+    <div v-if="newestVersion" class="note versionnote">
+      There is a newer version of this file available, the newest version can be found
```

Comment:
Tämä teksti on hyvin eksakti, mutta myös aika pitkä. Olisiko keinoa sanoa sama jotenkin napakammin? Esim. Zenodossa käytetään vastaavassa vain "There is a **newer version** of this record available." jossa linkki vie kaikkein uusimpaan recordiin. Koska teksti on myös Zenodon tapauksessa samanlaisessa pop-up infolaatikossa, se on minusta selvä että se vie uusimpaan versioon.

Harkitsitko myös muita värejä pohjaksi? Itselle tulisi mieleen, että tämmöinen info on vähän niin kuin varoitus ja pitäisi olla vaikka keltaisella, mutta ei tuo vihreäkään huonolta kyllä näytä.